lib/9pfs/9pfs.h

```diff
@@ -39,63 +39,150 @@
 
 #include <vfscore/prex.h>
 
-/*
- * Currently supports only the 9P2000.u variant of the protocol.
+/**
+ * Protocol version; currently only the `9P2000.u` variant of the protocol is supported
  */
 enum uk_9pfs_proto {
+	/*
+	 * Extensions and minor adjustments are added to
+	 * the basic protocol for Unix systems
+	 */
 	UK_9P_PROTO_2000U,
+	/*
+	 * Extensions and minor adjustments are added to
+	 * the basic protocol for Linux systems
+	 */
 	UK_9P_PROTO_2000L,
+	/*
+	 * The number of existent protocols;
+	 * in this case, two (`UK_9P_PROTO_2000U` and `UK_9P_PROTO_2000L`)
+	 */
 	UK_9P_PROTO_MAX
 };
 
+/**
+ * An entry containing the necessary data for mounting the filesystem
+ */
 struct uk_9pfs_mount_data {
-	/* 9P device. */
+	/* 9P device, used to interact with a `virtio` device;
+	 * the initialization for this field is done in
+	 * [`uk_9p_dev_connect`](https://github.com/unikraft/unikraft/blob/staging/lib/uk9p/9pdev.c#L237)
+	 */
 	struct uk_9pdev		*dev;
-	/* Wanted transport. */
+	/* Wanted transport */
 	struct uk_9pdev_trans	*trans;
-	/* Protocol version used. */
+	/* Protocol version used */
 	enum uk_9pfs_proto	proto;
-	/* Username to attempt to mount as on the remote server. */
+	/* User name to attempt to mount as on the remote server */
 	char			*uname;
-	/* File tree to access when offered multiple exported filesystems. */
+	/*
+	 * File system name to access when the server is
+	 * offering several exported file systems.
+	 */
 	char			*aname;
 };
 
+/**
+ * An entry containing the necessary data
+ */
 struct uk_9pfs_file_data {
-	/* Fid associated with the 9pfs file. */
+	/* File id of the 9pfs file */
 	struct uk_9pfid        *fid;
 	/*
 	 * Buffer for persisting results from a 9P read operation across
-	 * readdir() calls.
+	 * `readdir()` calls
 	 */
 	char                   *readdir_buf;
 	/*
 	 * Offset within the buffer where the stat of the next child can
```

Comment:
Add a reference to the stat structure so it's more clear.
```suggestion
	 * Offset within the buffer where the `uk_9p_stat` stat structure of the next child can
```